docs/guides/typescript.md

```diff
@@ -52,6 +52,63 @@ export const MyApp: React.SFC<{}> = () => {
 };
 ```
 
+#### Component props (`<Formik />` and `<FieldArray />`)
+
+```typescript
+import * as React from 'react'
+import { Field, FieldArray, Form, Formik, FormikActions } from 'formik'
+
+interface MyFormValues {
+  friends: string[]
+}
+
+export const FriendList = () => (
+  <div>
+    <h1>Friend List</h1>
+    <Formik
+      initialValues={{
+        friends: ['jared', 'ian', 'brent', 'beau'],
+      }}
+      onSubmit={(
+        values: MyFormValues,
+        actions: FormikActions<MyFormValues>
+      ) => {
+        console.log({ values, actions })
+        alert(JSON.stringify(values, null, 2))
+        actions.setSubmitting(false)
+      }}
+    >
+      {({ isSubmitting }) => (
+        <Form>
+          <FieldArray name="friends" component={FriendsFieldArray} />
+          <button type="submit" disabled={isSubmitting}>
+            Save
+          </button>
+        </Form>
+      )}
+    </Formik>
+  </div>
+)
+
+const FriendsFieldArray = (arrayHelpers: FieldArrayRenderProps) => (
+  <div>
+    {arrayHelpers.form.values[arrayHelpers.name].map(
```

Comment:
Using arrow function is 👍 , personally I would store values in readable var instead. No big deal.

```
const FriendsFieldArray = (arrayHelpers: FieldArrayRenderProps) => {
  const values = getIn(arrayHelpers.form.values, arrayHelpers.name);
  return values.map(...)
}
```

Using `getIn` allows you use "dot" patch, like `user.firstName`.

Reply:
Nice! Thanks for the tip about `getIn()` method. I hadn't seen this before. Will update PR.